modules/ROOT/pages/introduction.adoc

```diff
@@ -113,71 +105,7 @@ Once that happens, the migration is complete.
-Click the components to open a larger view.
-
-[.swiper]
-====
-[.slide]
---
-.Walk through the illustrated migration phases
-Discover the migration concepts, software components, and sequence of operations.
-
-image:migration-introduction9.png["Introductory page prompts you to click the Start button to begin the graphical presentation."]
---
-
-[.slide]
---
-.Phase 0: Before your migration starts
-Let's look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.
-
-image:pre-migration0ra9.png["Illustrates a pre-migration environment, as summarized in the text. Back and Next buttons are available for navigation within the graphic."]
---
-
-[.slide]
---
-.Phase 1: Deploy ZDM Proxy and connect client applications
-In this first phase, deploy the ZDM Proxy instances and connect client applications to the proxies. This phase activates the dual-write logic. Writes are bifurcated (sent to both Origin and Target), while reads are executed on Origin only.
-
-image:migration-phase1ra9.png["Illustrates migration Phase 1, as summarized in the text. Back and Next buttons are available for navigation within the graphic."]
---
-
-[.slide]
---
-.Phase 2: Migrate data
-In this phase, migrate existing data using Cassandra Data Migrator and/or DSBulk Migrator. Validate that the migrated data is correct, while continuing to perform dual writes.
-
-image:migration-phase2ra9.png["Illustrates migration Phase 2, as summarized in the text. Back and Next buttons are available for navigation within the graphic."]
---
-
-[.slide]
---
-.Phase 3: Enable asynchronous dual reads
-In this phase, you can optionally enable asynchronous dual reads. The idea is to test performance and verify that Target can handle your application's live request load before cutting over from Origin to Target.
-
-image:migration-phase3ra9.png["Illustrates migration Phase 3, as summarized in the text. Back and Next buttons are available for navigation within the graphic."]
---
-
-[.slide]
---
-.Phase 4: Route reads to Target
-In this phase, read routing on the ZDM Proxy is switched to Target so that all reads are executed on it, while writes are still sent to both clusters. In other words, Target becomes the primary cluster.
-
-image:migration-phase4ra9.png["Illustrates migration Phase 4, as summarized in the text. Back and Next buttons are available for navigation within the graphic."]
---
-
-[.slide]
---
-.Phase 5: Connect directly to Target
-In this phase, move your client applications off the ZDM Proxy and connect the apps directly to Target. Once that happens, the migration is complete.
-
-image:migration-phase5ra9.png["Illustrates migration Phase 5, as summarized in the text. Back and Restart buttons are available for navigation within the graphic."]
---
-====
-////
```

Comment:
would this change remove the infographic?

Reply:
This has been removed.

Reply:
Is there a reason why we would remove the infographic?